# Camera Pose Co-ordianates 
geometry_msgs/Pose camera_pose

# Essential matrix [3x3] 
float64[9] essential_matrix